src/main/java/org/onebusaway/csv_entities/schema/AbstractEntitySchemaFactoryImpl.java

```diff
@@ -400,18 +400,7 @@ private FieldMapping getFieldMapping(Class<?> entityClass, Field field,
       DefaultFieldMapping m = new DefaultFieldMapping(entityClass,
           csvFieldName, objFieldName, objFieldType, required);
 
-      try {
-        String name = field.getName();
-        String isFieldSet = "is" + Character.toUpperCase(name.charAt(0))
-            + name.substring(1) + "Set";
-        Method method = entityClass.getMethod(isFieldSet);
-        if (method != null
-            && (method.getReturnType() == Boolean.class || method.getReturnType() == Boolean.TYPE)) {
-          m.setIsSetMethod(method);
-        }
-      } catch (Exception ex) {
-        // We ignore this
-      }
+      setIsSetMethod(entityClass, field, m);
```

Comment:
I think this method call is redundant. `setIsSetMethod` will be called on the same object on line 419 (since `DefaultFieldMapping` extends `AbstractFieldMapping` and the next line sets `mapping = m`).

Reply:
Yes, good catch as to the control flow.  It doesn't make much sense for that code to be extracted out into a method of its own now, so I'll re-inline it.